From 6f8455ca64801c9b6b422f18942852e6b0ba8e32 Mon Sep 17 00:00:00 2001
From: Veera H <veera.h@adlinktech.com>
Date:   Tue Mar 5 10:42:34 2024 +0530

Subject: [PATCH 1/2] LEC-IMX8MP: Sema i2c driver remove function proto change

diff --git a/driver/adl-bmc.c b/driver/adl-bmc.c
index 6f796d9..3045712 100644
--- a/driver/adl-bmc.c
+++ b/driver/adl-bmc.c
@@ -177,7 +177,7 @@ static int adl_bmc_probe ( struct i2c_client *client, const struct i2c_device_id
 
 }
 
-static int adl_bmc_remove ( struct i2c_client *client) 
+static void adl_bmc_remove ( struct i2c_client *client) 
 {
 
 	struct device *dev = &client->dev;
@@ -186,7 +186,6 @@ static int adl_bmc_remove ( struct i2c_client *client)
 
 	mfd_remove_devices (dev);
 
-	return 0;
 }
 
 
